From 0b57892b0cb8cfc39f35d14efb37e7814497a8d4 Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@xxxxx.com>
Date: Wed, 24 Jan 2024 06:00:27 -0500
Subject: [PATCH] configs: odroid-n2_defconfig: `enable preboot with sleep`

CONFIG_USE_PREBOOT=y
CONFIG_PREBOOT="usb start; sleep 2"

The sleep helps to improve USB boot reliability

Signed-off-by: Patrick Yavitz <pyavitz@xxxxx.com>
---
 configs/odroid-n2_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/odroid-n2_defconfig b/configs/odroid-n2_defconfig
index e55de380a4..bd5ae3a7cc 100644
--- a/configs/odroid-n2_defconfig
+++ b/configs/odroid-n2_defconfig
@@ -18,6 +18,8 @@ CONFIG_SYS_LOAD_ADDR=0x1000000
 CONFIG_DEBUG_UART=y
 CONFIG_REMAKE_ELF=y
 CONFIG_OF_BOARD_SETUP=y
+CONFIG_USE_PREBOOT=y
+CONFIG_PREBOOT="usb start; sleep 2"
 # CONFIG_DISPLAY_CPUINFO is not set
 CONFIG_MISC_INIT_R=y
 CONFIG_SYS_MAXARGS=32
-- 
2.39.2

